docs/versioning.md

```diff
@@ -0,0 +1,112 @@
+
+The *<windows_build>* refers to the Windows SDK build number. As of .NET5 RTM, the supported *<windows_build>* versions are 17763, 18362, and 19041.
+
+The guidelines for incrementing `AssemblyVersion` are as follows. Note that `AssemblyVersion` increments to either assembly are independent of one another.
+
+1. For *WinRT.Runtime.dll*:
+    * Do *not* increment: if we only have backwards compatible bug-fixes.
+        * **Note**: this differs from Semantic Versioning rules
+    * Increment *\<minor>* or *\<major>*: if there are any changes to the public API surface.
+2. For *Microsoft.Windows.SDK.NET.dll*:
+    * Increment *\<projection_update>*: if there are changes to the CsWinRT generated API surface for the Windows SDK projections, for example if we expose more functions or if we discover we are not projecting a function that we should be.
+    * Update *\<windows_build>*: if there is a new version of the Windows API surface, which would occur with a new Windows OS release and corresponding SDK. This will involve listing a new SDK projection package for the new Windows version.
+
+### AssemblyFileVersion
+
+`AssemblyFileVersion` is used for deployment and can mark assemblies with the same AssemblyVersion. It can be viewed in Windows from **File Properties** -> **File Version**. `AssemblyFileVersion` has the following format for CsWinRT generated assemblies:
+
+| | AssemblyFileVersion Format | 
+|-|-|
+| **WinRT.Runtime.dll** | *\<major>.\<minor>.\<patch>.\<build>* |
+| **Microsoft.Windows.SDK.NET.dll** |  *10.0.<windows_build>.<package_update>* |
+
+`AssemblyFileVersion` will be incremented with each release, regardless of whether there are API changes. This means if `AssemblyVersion` does not increment, we will still increment `AssemblyFileVersion`. The guidelines for incrementing `AssemblyFileVersion` are as follows.
+
+1. For *WinRT.Runtime.dll*:
+    * Increment *\<minor>* or *\<major>*: if there is an increment to `AssemblyVersion`. This update would be in correspondence with the update to `AssemblyVersion`.
+    * Increment *\<patch>* and/or *\<build>*: if there is no increment to `AssemblyVersion`.
+2. For *Microsoft.Windows.SDK.NET.dll*:
+    * Increment *\<package_update>*.
+
+## Future Version Numbers
+
+As of .NET5 RTM (5.0.100), `AssemblyVersion` and `AssemblyFileVersion` for the runtime and SDK assemblies are identical. Going forwards, `AssemblyVersion` and `AssemblyFileVersion` will be decoupled.
+
+The CsWinRT NuGet version and SDK projection package version will correspond with the `AssemblyFileVersion` of *WinRT.Runtime.dll* and *Microsoft.Windows.SDK.NET.dll*, respectively.
+
+The table below is an example of how we will increment future version numbers. The **AssemblyVersion** and **AssemblyFileVersion** column cells list the *WinRT.Runtime.dll* version followed by the *Microsoft.Windows.SDK.NET.dll* version.
```

Comment:
could combine the last two columns for consistency, label the result "Nuget Package version", and update the legend: The version column cells list the WinRT.Runtime.dll/CsWinRT versions followed by the Microsoft.Windows.SDK.NET.dll/SDK versions.